.github/workflows/build-lint-test.yml

```diff
@@ -0,0 +1,73 @@
+name: Build, Lint, and Test"
+
+on:
+  workflow_call:
+
+jobs:
+  prepare:
+    name: Prepare
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: '.nvmrc'
+          cache: 'yarn'
+      - name: Install Yarn dependencies
+        run: yarn --immutable
```

Comment:
This is Yarn v3 syntax. There are a few other examples of this throughout as well.

Usually we do this in the opposite order - Yarn v3 upgrade first, then update GitHub actions to match the template.

Would it makes sense to switch the order here as well by rebasing the other PR onto main?

Reply:
@Gudahtt yes either I can switch the order or we can merge the Yarn v3 upgrade PR in, but I understand that would be a less transparent paper trail.

Reply:
Ok just swapped around the order. Now this is on top of https://github.com/MetaMask/eth-trezor-keyring/pull/150 which does the yarn v3 migration first.